Clarify the correlation between mTLS client cert and application id. (#13496)
digital-asset/daml: docs/source/app-dev/authorization.rst

They check for every Ledger API request whether the request contains an access token that is valid and sufficient to authorize the request.
You thus need to add support for authorization using access token to your application to run it against a deployed ledger.

For the

> .. note:: In case of mutual (two-way) TLS authentication setup the Ledger API

Reviewer: ```suggestion
.. note:: In case of mutual (two-way) TLS authentication, the Ledger API
```

>           client presents in addition to an access token its certificate to

Reviewer: ```suggestion
          client must present (in addition to an access token) its certificate to
```

>           client must present (in addition to an access token) its certificate to

Reviewer 2: ```suggestion
          client must present its certificate (in addition to an access token) to
```

>           Ledger API Server as part of the authentication process. The provided

Reviewer: ```suggestion
          the Ledger API server as part of the authentication process. The provided
```

          certificate must be signed by a certificate authority (CA) trusted

>           by Ledger API server. Note that the identity of the application

Reviewer: ```suggestion
          by the Ledger API server. Note that the identity of the application
```

>           will not be proven by using this method, i.e. application identity (application id)
>           is not necessarily correlated with the CN (Common Name) in the certificate.

Reviewer: ```suggestion
          will not be proven by using this method, i.e. the `application_id` field in the request
          is not necessarily correlated with the CN (Common Name) in the certificate.
```

.. note:: All Daml ledgers also support a deprecated legacy format of custom Daml claims
   access tokens whose format is equal to the above except for the custom claims
   to be present at the same level as ``exp`` in the token above,

>    instead of being nested below ``"https://daml.com/ledger-api"``.

Reviewer: Weird. What changed on this line?

Author: There was no newline in the end. My Intellij adds it automatically. Shall I remove it?

Reviewer: Nah, thanks, I just couldn't figure out what changed. Thanks. 🙂
